website/static/js/logFeed.js

```diff
@@ -1,238 +1,245 @@
-      * Given an item in self.contributors, return its anchor element representation.
-      */
-    self._asContribLink = function(person) {
-        return '<a class="contrib-link" href="/profile/' + person.id + '/">' +  $osf.htmlEscape(person.fullname) + '</a>';
-    };
-
-    self.hasUser = ko.pureComputed(function() {
-        return Boolean(self.user && self.user.fullname);
-    });
-
-    /**
-      * Return whether a knockout template exists for the log.
-      */
-    self.hasTemplate = ko.computed(function() {
-        if (!self.hasUser()) {
-            return $('script#' + self.action + '_no_user').length > 0;
-        } else {
-            return $('script#' + self.action).length > 0;
-        }
-    });
+var MAX_PAGES_ON_PAGINATOR = 7;
+var MAX_PAGES_ON_PAGINATOR_SIDE = 5;
 
-    self.mapUpdates = function(key, item) {
-        if (key === 'category') {
-            return key + ' to ' + nodeCategories[item['new']];
-        }
-        else {
-            return key + ' to ' + item;
-        }
-    };
-
-    /**
-      * Return the html for a comma-delimited list of contributor links, formatted
-      * with correct list grammar.
-      * e.g. "Dasher and Dancer", "Comet, Cupid, and Blitzen"
-      */
-    self.displayContributors = ko.computed(function(){
-        var ret = '';
-        if (self.anonymous){
-            ret += '<span class="contributor-anonymous">some anonymous contributor(s)</span>';
-        } else {
-            for (var i = 0; i < self.contributors.length; i++) {
-                var person = self.contributors[i];
-                if (i === self.contributors.length - 1 && self.contributors.length > 2) {
-                    ret += ' and ';
-                }
-                if (person.registered) {
-                    ret += self._asContribLink(person);
-                } else {
-                    ret += '<span>' + $osf.htmlEscape(person.fullname) + '</span>';
-                }
-                if (i < self.contributors.length - 1 && self.contributors.length > 2) {
-                    ret += ', ';
-                } else if (i < self.contributors.length - 1 && self.contributors.length === 2) {
-                    ret += ' and ';
-                }
-            }
+var LOG_PAGE_SIZE_LIMITED = 3;
+var LOG_PAGE_SIZE = 6;
+
+/* Send with ajax calls to work with api2 */
+var xhrconfig = function (xhr) {
+    xhr.withCredentials = window.contextVars.isOnRootDomain;
+    xhr.setRequestHeader('Content-Type', 'application/vnd.api+json;');
+    xhr.setRequestHeader('Accept', 'application/vnd.api+json; ext=bulk');
+};
+
+var _buildLogUrl = function(node, page, limitLogs) {
+    if (!node.isRetracted || !node.is_retracted) {
+        var urlPrefix = (node.isRegistration || node.is_registration) ? 'registrations' : 'nodes';
+        var size = limitLogs ? LOG_PAGE_SIZE_LIMITED : LOG_PAGE_SIZE;
+        var query = { 'page[size]': size, 'page': page, 'embed': ['original_node', 'user', 'linked_node', 'template_node']};
```

Comment:
Does our API gracefully handle `page=undefined`? It looks like undefined will be passed [here](https://github.com/CenterForOpenScience/osf.io/blob/938ecbdff7bc53c19bfa87c1cce8b5412e830b1c/website/static/js/logFeed.js#L93). 

Even if it is handled by the API, it would be worth passing `page=1` if page isn't passed, for readability.

```js
page = page || 1;
```